src/main/java/com/juniorstart/juniorstart/model/JobOfferRequirements.java

```diff
@@ -0,0 +1,25 @@
+package com.juniorstart.juniorstart.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import javax.persistence.*;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class JobOfferRequirements {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @NotNull
+    private String textRequirement;
+
+    @JsonIgnore
+    @ManyToOne(fetch = FetchType.LAZY, optional = true, cascade =  CascadeType.ALL)
+    @JoinColumn(name = "jobOffer_id", nullable = true)
+    private JobOffer jobOffer;
```

Comment:
Nie lepiej zrobić relację jednostronną? JobOfferRequirements chyba nie musi wiedzieć o tym jakie oferty go posiadają

Reply:
@Ejden lepiej tak zostawić będzie łatwiej pisać metody w serwisie, które wyciągają jobOfferRequirements po jobogffer

Reply:
Dodał bym jeszcze tutaj usera

Reply:
Nie lepiej zrobić tak, że jobOfferRequirements pobierają się razem z JobOffer? Bo jak ktoś będzie chciał pobrać jobOffer to raczej chcę pobrać całość na raz. Więc nie trzeba też później nawet działać na tej tabeli.